sechub-testframework/src/main/java/com/daimler/sechub/test/TestURLBuilder.java

```diff
@@ -301,6 +301,10 @@ public String buildAdminListsProjectsUrl() {
     public String buildAdminShowsProjectDetailsUrl(String projectId) {
         return buildUrl(API_ADMIN_PROJECT, projectId);
     }
+    
+    public String buildAdminChangesProjectDescriptionUrl(String projectId) {
```

Comment:
In my opinion, this should be called `buildAdminChangesProjectUrl`.

Reply:
done